app/views/search/bento.html.erb

```diff
@@ -1,29 +1,46 @@
 <%= render partial: "form" %>
 
-<div class="col-sm-4">
-  <h2>Articles and Stuff</h2>
-  <div id="articles_content">
-    Loading results...
-    <i class="fa fa-spinner fa-spin"></i>
+<div class="row">
+  <div class="col-sm-4">
+    <h2>Articles and Stuff</h2>
+    <div id="articles_content">
+      Loading results...
+      <i class="fa fa-spinner fa-spin"></i>
+    </div>
   </div>
-</div>
 
-<div class="col-sm-4">
-  <h2>Books and Stuff</h2>
-  <div id="books_content">
-    Loading results...
-    <i class="fa fa-spinner fa-spin"></i>
+  <div class="col-sm-4">
+    <h2>Books and Stuff</h2>
+    <div id="books_content">
+      Loading results...
+      <i class="fa fa-spinner fa-spin"></i>
+    </div>
   </div>
-</div>
 
-<div class="col-sm-4">
-  <h2>Website and Guides</h2>
-  <div id="website_content">
-    Loading results...
-    <i class="fa fa-spinner fa-spin"></i>
+  <div class="col-sm-4">
+    <h2>Website and Guides</h2>
+    <div id="website_content">
+      Loading results...
+      <i class="fa fa-spinner fa-spin"></i>
+    </div>
   </div>
 </div>
 
+<div class="row">
+  <h2>Not finding what you're looking for? Try these other tools:</h2>
+  <dl>
+    <dt><a href="//libraries.mit.edu/vera">Vera</a></dt>
+    <dd>Best for e-journals and databases owned or licensed by the MIT Libraries. Don’t search for individual articles here.</dd>
+    <dt><a href="//libraries.mit.edu/worldcat">WorldCat</a></dt>
+    <dd>Best for books and other items not owned by the MIT Libraries. Using this interface, you can request items from <a href="//libraries.mit.edu/borrow-direct">Borrow Direct</a> or <a href="//libraries.mit.edu/ilb">Interlibrary Borrowing (ILB)</a>.</dd>
+  </dl>
+  <h3>Specialized search tools &amp; guides</h2>
```

Comment:
Yeah, I grabbed a section of content from libraries.mit.edu/search as a conversation starter - I'm curious what the response will be.